packages/react-native/Libraries/AppDelegate/RCTAppDelegate+Protected.h

```diff
@@ -8,9 +8,9 @@
 #if defined(__cplusplus)
 
 #import <ReactCommon/RCTTurboModuleManager.h>
-#import "RCTAppDelegate.h"
+#import "RCTReactNativeFactory.h"
 
-@interface RCTAppDelegate () <RCTTurboModuleManagerDelegate>
+@interface RCTReactNativeFactory () <RCTTurboModuleManagerDelegate>
 @end
```

Comment:
Im not sure how to handle this case, any ideas @cipolleschi ?

Reply:
uhm... this is something we use internally I forgot about... Who is implementing the methods from `RCTTurboModuleManager` with these changes?

Reply:
Looks like the delegate of ReactNativeFactory needs to implement this to be able to easily override it. Probably this file can be re-named but we need to keep the `#if defined(__cplusplus)` to make sure Swift keeps working after these changes